### ISSUE TITLE
config: Create a configuration.toml for Kata

### ISSUE THREAD

**Issue by sboeuf** [clearcontainers/runtime#920]
We should provide an alternate `configuration_kata.toml.in` file in order to generate `configuration_kata.toml`. This would prevent from confusion if we want to test Kata and Clear on the same machine.

**Comment by jodh-intel** [comment]
If we are careful with how we handle #921, we could set an extra make variable to be either `kata` or `cc` or something and then conditionally set `CONFIG_FILE` rather than the current:

```
CONFIG_FILE = configuration.toml
```

**Comment by jodh-intel** [comment]
Hi @sboeuf - I'm now thinking it would be better to generate a single `configuration.toml` for a particular `PROJECT_NAME` (see #921). If you agree, can we close this issue (or update it to generate a `configuration.toml` for a kata system rather than actually create a `configuration-kata.toml`)?

**Comment by sboeuf** [comment]
@jodh-intel yes I agree this would make more sense, and eventually, this config file would be copied to the right directory `/etc/share/defaults/kata-containers/configuration.toml` when calling into `make kata-install`.
